ambry-account/src/test/java/com/github/ambry/account/MySqlAccountServiceTest.java

```diff
@@ -192,8 +189,9 @@ public void testBackgroundUpdater() throws InterruptedException, SQLException, I
 
     // force shutdown background updater thread. As the default timeout value is 1 minute, it is possible that thread is
     // present after close() due to actively executing task.
-    ((MySqlAccountService) mySqlAccountService).getScheduler().shutdownNow();
+    mySqlAccountService.getScheduler().shutdownNow();
 
+    Thread.sleep(100);
```

Comment:
@Arun-LinkedIn Added the sleep after the assertion failed.  Thread removal may not be immediate.

Reply:
For cases where a poll with timeout is needed, we have a test utility that can be used:

```
TestUtils.checkAndSleep(0, () -> numThreadsByThisName(MySqlAccountService.MYSQL_ACCOUNT_UPDATER_PREFIX), 200);
```

Reply:
Good point.  I'll change that in a follow up PR.